TM1py/Services/CellService.py

```diff
@@ -568,6 +568,55 @@ def clear(self, cube: str, **kwargs):
 
         return self.clear_with_mdx(cube=cube, mdx=mdx_builder.to_mdx(), **kwargs)
 
+    @require_data_admin
+    @require_ops_admin
+    @require_version(version="11.7")
+    def clear_from_df(self, cube_name: str, df: pd.DataFrame, dimension_mapping: Dict = None, **kwargs):
```

Comment:
Please add a docstring.

The `dimension_mapping` and the fact the deleted area will be the cross join of all selections is maybe not obvious to users.

Reply:
Done. Please let me know if there is anything else missing.